src/Xamarin.Android.Build.Tasks/Tasks/CalculateLayoutCodeBehind.cs

```diff
@@ -487,7 +487,7 @@ bool ParseID (string id, out string parsedId, out string name)
 			if (id.StartsWith ("@id/", StringComparison.Ordinal) || id.StartsWith ("@+id/", StringComparison.Ordinal))
 				ns = "Resource.Id";
 			else if (id.StartsWith ("@android:id/")) {
-				ns = "Android.Resource.Id";
+				ns = "global@Android.Resource.Id";
```

Comment:
No, it's a placeholder - language agnostic. I just use it to catch "global" (or fully-qualified) type names so that the generator doesn't prefix it with `global::` (or F#, VB etc equivalent)